observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemDefinition.java

```diff
@@ -40,54 +44,59 @@ class MicrometerSubsystemDefinition extends PersistentResourceDefinition {
     private static final String MICROMETER_MODULE = "org.wildfly.extension.micrometer";
     private static final String MICROMETER_API_MODULE = "org.wildfly.extension.micrometer-api";
```

Comment:
The Micrometer API module name is definitely wrong. This is not a WF extension in the sense we use this word. Also applies to java package name but I can't comment on that line since it's not being changed.

How about "org.wildfly.micrometer.deployment" ?

Reply:
To save anyone time who might be looking at this: this module contains single CDI extension class (MicrometerCdiExtension) which registers the MeterRegistry type.

Reply:
Changed.